.circleci/config.yml

```diff
@@ -57,6 +56,18 @@ jobs:
           name: Publish Docker Image to Docker Hub
           command: |
             echo "${DOCKERHUB_PASS}" | docker login -u "${DOCKERHUB_USERNAME}" --password-stdin
+            TAG=${CIRCLE_TAG}
+            D_TAG="$(wget -q https://registry.hub.docker.com/v1/repositories/logdna/logdna-agent/tags -O - | jq -r '.[] | .name' | awk '!/-/ && /\./ { print $1}' | grep ${MAJOR_VERSION} | tail -1)"d
```

Comment:
There is a `d` at the end of this line. Copy pasta or go issue I think

Reply:
maybe it's intentional? the var name is D_TAG and this certainly would put a d character in the tag string...